pkg/reconciler/pipelinerun/resources/validate_params_test.go

```diff
@@ -19,159 +19,133 @@ package resources
 import (
 	"testing"
 
-	tb "github.com/tektoncd/pipeline/internal/builder/v1beta1"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func TestValidateParamTypesMatching_Valid(t *testing.T) {
-	tcs := []struct {
-		name          string
-		p             *v1beta1.Pipeline
-		pr            *v1beta1.PipelineRun
-		errorExpected bool
+
+	mockStringValue := *v1beta1.NewArrayOrString("mockStringValue")
+	mockArrayValue := *v1beta1.NewArrayOrString("mockArrayValue", "mockArrayValue")
```

Comment:
nit: technically, this is not a "mock" :stuck_out_tongue_closed_eyes: we could name it `stringValue` or anything.
But it's a nit, no need to fix it now :wink:

Reply:
```bash
%s/mockStringValue/stringValue/g
%s/mockArrayValue/arrayValue/g
```